server/methods/core/groups.js

```diff
@@ -0,0 +1,207 @@
+import _ from "lodash";
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Roles } from "meteor/alanning:roles";
+import { Logger } from "/server/api";
+import { Accounts, Shops } from "/lib/collections";
+
+/**
+ * Reaction Group Permission Methods
+ * TODO: Confirm if better to move this to shop.js since groups are on shops
+ */
+Meteor.methods({
+  /**
+   * group/createGroup
+   * @summary creates a new permission group for a shop
+   * creates permission group for a given shop with passed in roles
+   * @param {Object} groupData - info about group to create
```

Comment:
Looks good. Might be worth having some meta information on each group as well, what do you think?

E.g. createdAt, updatedAt, createdBy: username, etc